app/components/phase_banner_component.html.erb

```diff
@@ -1,10 +1,11 @@
-<div class="app-phase-banner<%= @no_border ? ' app-phase-banner--no-border' : '' %>">
+<div class="app-phase-banner<%= @no_border ? ' app-phase-banner--no-border' : '' %> govuk-!-display-none-print">
   <div class="govuk-width-container">
-    <div class="govuk-phase-banner govuk-!-display-none-print">
-      <p class="govuk-phase-banner__content">
-        <%= govuk_tag(text: HostingEnvironment.phase, classes: "govuk-phase-banner__content__tag #{phase_tag_class}") %>
-        <span class="govuk-phase-banner__text"><%= text %></span>
-      </p>
-    </div>
+    <%= govuk_phase_banner(
+      phase_tag: {
+        text: HostingEnvironment.phase,
+        colour: HostingEnvironment.phase_colour,
```

Comment:
@duncanjbrown This is not going to return a value if the hosting environment is `production`. Will this break anything (and how can I test that?)

Reply:
I’m guessing it will return `nil`? In which case, nothing will break.

Reply:
`nil` is safe here — you can see the `phase_banner_component_spec` already runs itself in "Production" by setting an environment variable in an `around` block

Reply:
Cool, will merge.